frontend/src/components/form/WorkoutForm.tsx

```diff
@@ -0,0 +1,172 @@
+import {useNavigate} from "react-router-dom";
+import React, {useEffect, useState} from "react";
+import {Workout, WorkoutExercise, WorkoutNoId} from "../../types/types.ts";
+import axios from "axios";
+
+type WorkoutFormType = {
+    formType: "new" | "edit",
+    onWorkoutChange: () => void,
+    initialWorkout: Workout
+}
+
+function WorkoutForm({formType, initialWorkout, onWorkoutChange}: Readonly<WorkoutFormType>) {
```

Comment:
```suggestion
function WorkoutForm(props: Readonly<WorkoutFormType>) {
```
Destructing nicht nötig, da du eh alles benutzt

Reply:
Ja, allerdings möchte ich nicht props davor schreiben und ausschließlich über den variablen Namen darauf zugreifen. Ist dieser Ansatz bad practice?